assets/js/src/pwa.js

```diff
@@ -7,6 +7,19 @@
     window.addEventListener('load', function () {
       navigator.serviceWorker.register('/sw.js').then(function (registration) {
         console.log('ServiceWorker registration successful with scope: ', registration.scope)
+        registration.onupdatefound = () => {
+          var installingWorker = registration.installing;
+          installingWorker.onstatechange = () => {
```

Comment:
Can you use ES5 here? We use uglify-js which doesn't support ES6 (there's a branch, I know, but we don't use ES6 here).